statsrunner/loop.py

```diff
@@ -26,28 +38,46 @@ def call_stats(this_stats, args):
 
 
 def process_file((inputfile, output_dir, folder, xmlfile, args)):
+    """Create output file path or write output file."""
     import importlib
+    # Python module to import stats from defaults to stats.dashboard
     stats_module = importlib.import_module(args.stats_module)
-    
+    # When args.verbose_loop is true, create directory and set outputfile
+    # according to loop path.
     if args.verbose_loop:
-        try: os.makedirs(os.path.join(output_dir,'loop',folder))
-        except OSError: pass
-        outputfile = os.path.join(output_dir,'loop',folder,xmlfile)
+        try:
+            os.makedirs(os.path.join(output_dir, 'loop', folder))
+        except OSError:
+            pass
+        outputfile = os.path.join(output_dir, 'loop', folder, xmlfile)
+    # If args.verbose_loop is false, set outputfile according to aggregated-file
+    # path.
     else:
-        outputfile = os.path.join(output_dir,'aggregated-file',folder,xmlfile)
+        outputfile = os.path.join(output_dir, 'aggregated-file', folder, xmlfile)
 
+    # If default args is set to only create new files, check for existing file
+    # and return.
```

Comment:
Returning early (I assume from my understanding of the code). Have now specified in the comments.